impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -124,7 +124,10 @@ bool RuntimeEffectContents::Render(const ContentContext& renderer,
   desc.SetVertexDescriptor(std::move(vertex_descriptor));
   desc.SetColorAttachmentDescriptor(
       0u, {.format = color_attachment_format, .blending_enabled = true});
-  desc.SetStencilAttachmentDescriptors({});
+
+  StencilAttachmentDescriptor stencil0;
+  stencil0.stencil_compare = CompareFunction::kEqual;
```

Comment:
The stencil depth (you can get this from the `Entity`) should also be set here, otherwise any clips will cause the output to get discarded.

Reply:
Specifically, the "stencil_reference" field.

Reply:
Ah whoops, I missed that we're already setting it.

The reason we have the stencil ref on the command and not the pipeline descriptor is that it's a pipeline input, not a property that we need to hash and build new pipelines for.